ui/apps/platform/src/Containers/MainPage/MainPage.tsx

```diff
@@ -51,6 +63,17 @@ function MainPage(): ReactElement {
     const { isFeatureFlagEnabled, isLoadingFeatureFlags } = useFeatureFlags();
     const { hasReadAccess, hasReadWriteAccess, isLoadingPermissions } = usePermissions();
 
+    // Check for clusters under management
+    // if none, and user can admin Clusters, redirect to clusters section
```

Comment:
The phrase **and use can admin Clusters** implies a potential timing problem, because the following `if` statement waits until response from /v1/mypermissions request.